source/scripting_v3/GTA/World.cs

```diff
@@ -2116,6 +2150,26 @@ public static RelationshipGroup AddRelationshipGroup(string name)
 			return new RelationshipGroup(resultArg);
 		}
 
+		/// <summary>
+		/// Creates a shocking event of specified type at the given position for the given duration.
+		/// </summary>
+		/// <param name="eventType">The type of shocking event peds nearby will perceive.</param>
+		/// <param name="position">The epicenter position of the shocking event.</param>
+		/// <param name="duration">The duration of the shocking event.</param>
+		public static void CreateShockingEvent(EventType eventType, Vector3 position, float duration)
+			=> Function.Call(Hash.ADD_SHOCKING_EVENT_AT_POSITION, (int)eventType, position.X, position.Y, position.Z, duration);
+
+		/// <summary>
+		/// Suppresses shocking events per frame, preventing all peds from reacting to them.
+		/// </summary>
+		public static void SuppressShockingEventsNextFrame() => Function.Call(Hash.SUPPRESS_SHOCKING_EVENTS_NEXT_FRAME);
+
+		/// <summary>
+		/// Sets the intensity of the "shaking" effect of all vehicles at speed.
+		/// </summary>
+		/// <param name="multiplier">Intensity of the shaking effect between 0f and 1f.</param>
+		public static void VehicleShakeMultiplier(float multiplier) => Function.Call(Hash.SET_CAR_HIGH_SPEED_BUMP_SEVERITY_MULTIPLIER, multiplier);
```

Comment:
Because it was being used to remove the vehicle shake generally. I hadn't seen that mod, but BTTFV calls this at startup which is doing the same thing. I've no problem putting the name back to match the native though, was just trying to make it more concise.